### PR TITLE
v2raya.win version 1.5.6.2

### PR DESCRIPTION
**Pull request by MarksonHon**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/44154)

### CODE REVIEW

File: manifests/v/v2raya/win/1.5.6.2/v2raya.win.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: v2raya.win
+PackageVersion: 1.5.6.2
+PackageLocale: en-US
+Publisher: v2rayA Developers
```
Comment (ItzLevvie) [inline manifests/v/v2raya/win/1.5.6.2/v2raya.win.locale.en-US.yaml:7]:
```suggestion
Publisher: The v2rayA developer community
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/v/v2raya/win/1.5.6.2/v2raya.win.locale.en-US.yaml (hunk 1 above)
Comment (MarksonHon) [reply manifests/v/v2raya/win/1.5.6.2/v2raya.win.locale.en-US.yaml:7]: Thanks for your advise, This change is acceptable and good

Comment (ItzLevvie) [reply manifests/v/v2raya/win/1.5.6.2/v2raya.win.locale.en-US.yaml:7]: cc @MarksonHon: Would you be able to commit the changes on your end?

**Comment by ItzLevvie** [comment]
Would you be able to change the `Publisher` to
```suggestion
Publisher: The v2rayA developer community
```

Looks like GitHub is having lots of Server Errors on my end.

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/44154#issuecomment-1031688978

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**